src/Compilers/CSharp/Portable/Syntax/AnonymousMethodExpressionSyntax.cs

```diff
@@ -3,6 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 #nullable disable
+#pragma warning disable RS0041 // uses oblivious reference types
```

Comment:
There aren't that many APIs in these files. Should we just annotate them?

Reply:
I'd tried that but it snowballed the change. Admittedly this was a while back, but my recollection is some of those have many usage sites that needed adjustment. So I'd kept the PR focused on its primary purpose.

Reply:
Good enough reason for me.